Tech@HubAPI/Tech@HubAPI/Tech@HubAPI.csproj

```diff
@@ -12,13 +12,6 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
 
```

Comment:
The integration tests are failing because the `Profile_Pictures_JPG` folder isn't being included in the build output. Go ahead and add this to the Tech@HubAPI.csproj file

```
<ItemGroup>
      <Content Include="Profile_Pictures_JPG\**">
          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      </Content>
  </ItemGroup>
```

I think that should fix it